index.ts

```diff
@@ -213,6 +214,13 @@ addTests('isDiscussion', [
 	'https://github.com/orgs/community/discussions/11202',
 ]);
 
+export const isNewDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => Boolean(getRepo(url)?.path.startsWith('discussions/new') ?? getOrg(url)?.path.startsWith('discussions/new'))
```

Comment:
```suggestion
export const isNewDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => /^discussions\/new/.test(getRepo(url)?.path ?? getOrg(url)?.path!)
```
This will be more concise and consistent with the previous code style.
However, `getRepo(url)?.path ?? getOrg(url)?.path!` also appears in the `isDiscussion` function, maybe you can extract it into a new function for reuse.

Reply:
I think it’s a rare coincidence, not worth the deduplication

Reply:
> This will be more concise and consistent with the previous code style.

Thank for your suggest. But I think string(`'discussions/new'`) is more readable than regular expression(`/^discussions\/new/`), and this case is simple enough. So I'm will keeping the origin version.

---

And I use equals operator instead of `startsWith()` function, because equals operator is stricter. you can see [513af84](https://github.com/refined-github/github-url-detection/pull/177/commits/513af842ab722171a2fa88f5c60daeba97a90f59)